managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorContainer.java

```diff
@@ -319,6 +322,34 @@ private void swap(Item item1, Item item2) {
         item2.idx = idx1;
     }
 
+    private void updateSlowestReadPositionForActiveCursors(ManagedCursor managedCursor) {
+        if (managedCursor == null) {
+            PositionImpl slowest = null;
+            for (ManagedCursor cursor : this) {
+                PositionImpl p = (PositionImpl) cursor.getReadPosition();
+                if (slowest == null) {
+                    slowest = p;
+                } else if (p.compareTo(slowest) < 0) {
+                    slowest = p;
+                }
+            }
+            slowestReadPositionForActiveCursors = slowest;
+            return;
+        }
+        PositionImpl position = (PositionImpl) managedCursor.getReadPosition();
+        if (slowestReadPositionForActiveCursors == null) {
+            slowestReadPositionForActiveCursors = position;
+            return;
+        }
+        if (slowestReadPositionForActiveCursors.compareTo(position) > 0) {
+            slowestReadPositionForActiveCursors = position;
+        }
```

Comment:
If the read position of cursors moves forward, the `slowestReadPositionForActiveCursors` will not change?